Classes/Core.php

```diff
@@ -365,30 +365,16 @@ public static function registerFluidFlexFormTable(
      * @param string $providerExtensionName Vendor.ExtensionName format of extension scope of the template file
      * @param string $templateFilename Absolute path to template file containing Flux definition, EXT:... allowed
      * @param array $variables Optional array of variables which are assigned when rendering the Flux definition
-     * @param null|string $section
-     * @param null|string $paths
-     * @return ProviderInterface
      */
     public static function registerTemplateAsContentType(
         $providerExtensionName,
-        $templateFilename,
-        $variables = [],
-        $section = 'Configuration',
-        $paths = null
+        $templateFilename
```

Comment:
> has this been part of a stable release yet?

No, has only ever existed in `dev-development` - that's why I didn't care so much to preserve the API, I expect nobody is using it yet.